description.md

```diff
@@ -32,6 +32,80 @@ Put differently, it shows how each content type is to be used in the various tea
+  They consist of:
+
+  * [**Slides**](#slides) are the typical structured and simplified format for organizing ideas.
+    The lecturer walks through slides and presents information by expanding ideas on the slides, and by engaging in discussions based on those ideas.
+
+  * [**Media files**](#media) are generally images or videos that represent a concept in an easier to grasp format.
+    Diagrams, photos, recordings of actions / processes, recorded simulations fall under this category.
+    They can be used either stand-alone or embedded into slides.
+
+* **Practical support materials** are used to demonstrate actual real-world applications of presented concepts.
+  The lecturer is the one who uses the practical support materials to do live demonstrations and highlight concrete use cases.
+  This may not be possible for certain topics, either because they are highly theoretical or because it is difficult to have a live setup.
+  Where a live demonstration is not possible, a video recording (as a visual support material) may be used.
+  They consist of [**Demos**](#demos).
+
+* **Assessment materials** are used to evaluate knowledge and skills of participants, and to provide feedback on spots they need to focus more on.
+  Typically they are to be used live during lectures, with their results providing immediate feedback and triggering discussions.
+  They consist of [**Quizzes**](#quizzes).
+
+* **Offline self-study materials** are to be used by participants outside lectures.
+  They are detailed in the ["Self-Study" section](#self-study).
+
+As other types of content, lecture materials are publicly available, and also open to contributions.
+Lecture materials are meant to be used live by teachers during lectures and then offline by learners, to improve gained knowledge and skills.
+Visual support materials are directly available on the course website;
+the other materials are available either as downloadable archives or directly in the course repository.
+
+#### Lecture Content Structure
+
+Teachers have individual styles and preferences.
+Hence, they would define their prefered order of presenting topics and decide on the weight of each topic.
+
+Even so, it is recommended that the types of materials shown above be intertwined during an actual lecture.
+Slides are good for presenting general topics, but tend to become tern and difficult to follow.
+As such, spreading demos and quizzes between slides is great to engage the audience and also provide a practical overview of items and an insight into topics that deserve continued attention.
+
+One possibility is to have lectures focused primarily on demos and on quizzes and slides to be used sparringly, only as topic highlights.
+
+In the ideal case, each topic in a lecture will be a balanced mix of demos, quizzes, slides and media file.
+The dedicated teacher will be careful to adjust the types of content to the topic being presented and to the audience.
+
```

Comment:
You usually ditch slides altogether and only use demos and quizzes. You should present this approach as well. Maybe split them into subsections? Although the heading would be 5 `#`s, so it would probably be too much.

Either way, I would emphasize experimenting with various mixes according to the type of audience. Maybe people from different cultures react differently to "stimuli" such as quizzes or demos.

Reply:
Agree. Will augment that.